articles/api-auth/blacklists-vs-grants.md

```diff
@@ -0,0 +1,72 @@
+---
+description: This document covers blacklists vs. grants when it comes handling tokens.
+---
+
+# Blacklists and Client Grants
+
+Let's say that you're using a non-interactive [client](/client) to represent your API. You have a partner that calls your API, and at the end of your existing contract, you and your partner decide not to renew your partnership. As such, you now want to remove your partner's access to your API. The issue, however, is that you've given your partner an [access token](/tokens/access-token) that lasts for a month.
```

Comment:
@khorne3 The first sentence still looks wrong.You don't use a client to **represent** an API, you use a client to **access** an API (you use an API to represent a protected resource/API). I'm changing this myself, in case I got it wrong please send a fix.